### DISCUSSION TITLE
Esp32-Korvo Le micro qu’il te faut ! - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#27] [Comments]
# Esp32-Korvo Le micro qu’il te faut ! - Haade.fr

Esp32-korvo woover-e est le microphone complet pour contrôler la domotique par la voix dans home-assistant grâce à esphome et le wifi le tout pour moins de 30€

https://haade.fr/fr/blog/homeassistant-assist-esphome-esp32-korvo-wroover-e-microphone-array

**Comment by orrionis** [comment]
Bonjour il un bug dans votre config yaml
Pin 0 is used in multiple place
i2s_audio:
  - id: codec
    i2s_lrclk_pin: GPIO22
    i2s_bclk_pin: GPIO25
    i2s_mclk_pin: GPIO0
Pin 0 is used in multiple place
  - id: mic_adc
    i2s_lrclk_pin: GPIO26
    i2s_bclk_pin: GPIO27
    i2s_mclk_pin: GPIO0

en effet il senble que vous uti:lisez deux fois le meme GPIO

je suis sous la dernière version de ESPHome.

avez vous deja vue ce genre de problème ?
